pandas/tests/groupby/test_grouping.py

```diff
@@ -303,6 +303,18 @@ def test_grouper_getting_correct_binner(self):
     def test_grouper_iter(self, df):
         assert sorted(df.groupby('A').grouper) == ['bar', 'foo']
 
+    def test_grouper_base(self):
```

Comment:
can you move to test pandas/tests/resample/test_timegrouper and also test in .resample (which is what this calls)

Reply:
Changed it